From 7cf09b79d058fda07822a06726b606d8556bada3 Mon Sep 17 00:00:00 2001
From: Andrew Branson <andrew.branson@jollamobile.com>
Date: Wed, 10 Aug 2016 11:53:35 +0200
Subject: [PATCH] [tracker] Fix flac tag extraction. Fixes JB#35939

---
 src/tracker-extract/10-flac.rule | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tracker-extract/10-flac.rule b/src/tracker-extract/10-flac.rule
index 088e72285..4e1924364 100644
--- a/src/tracker-extract/10-flac.rule
+++ b/src/tracker-extract/10-flac.rule
@@ -1,4 +1,4 @@
 [ExtractorRule]
 ModulePath=libextract-flac.so
-MimeTypes=audio/x-flac
+MimeTypes=audio/flac;audio/x-flac
 FallbackRdfTypes=nfo:Audio;
-- 
2.28.0

